keanu/models/orders.py

```diff
@@ -4,8 +4,8 @@
 class Order(db.Document):
     config_collection_name = 'orders'
 
-    items = db.ListField(db.IntField(),required=True)
+    items = db.ListField(db.StringField(),required=True)
```

Comment:
`(db.StringField(),required=True)`

should be 

`d(db.StringField(), required=True)`